### DISCUSSION TITLE
disable auto update

### DISCUSSION THREAD

**Discussion by intuitive-arts-ai** [upscayl/upscayl#548] [Q&A] [answered]
Hi,
I'm sure updates will be great, but just in case, how do I disable the auto update?

Thanks!

**Comment by aaronliu0130** [comment, marked as answer]
![image](https://github.com/upscayl/upscayl/assets/11874211/fc2cdc3c-3190-43fc-b9e3-68a095e86394)
You can dismiss it by pressing "No Thanks"

**Comment by intuitive-arts-ai** [comment]
THanks a lot
